tests/test_camera_readout.py

```diff
@@ -128,11 +128,12 @@ def test_no_opsim(self):
         ccd = imsim.Camera(camera_name)[det_name]
         for amp in range(16):
             read_noise = list(ccd.values())[amp].read_noise
+            gain = list(ccd.values())[amp].gain
             amp_mean = np.mean(readout.final_data[amp+1].data)
             amp_var = np.var(readout.final_data[amp+1].data)
             assert np.isclose(amp_mean, 1000 + 0.02*dark_time, rtol=1.e-3)
-            print(amp_var, 0.02*dark_time + read_noise**2, (0.02*dark_time + read_noise**2-amp_var)/amp_var)
-            assert np.isclose(amp_var, 0.02*dark_time + read_noise**2, rtol=3.e-3)
+            print(amp_var, 0.02*dark_time/gain + read_noise**2, (0.02*dark_time/gain + read_noise**2-amp_var)/amp_var)
```

Comment:
Are we switching ow from ADUs to electrons here (now dividing by gain)?

Reply:
It's the other way around.  Dividing by the gain here converts from electrons to ADU.  This was always needed, but the gain values in the earlier weeklies of obs_lsst were sufficiently close to unity, that the test still passed.  The current gain values are now 1.5-1.7.